SU2_CFD/src/output_structure.cpp

```diff
@@ -8883,6 +8892,920 @@ void COutput::SetEquivalentArea(CSolver *solver_container, CGeometry *geometry,
   
 }
 
+void COutput::WriteSurface_Analysis(CConfig *config, CGeometry *geometry, CSolver *FlowSolver) {
```

Comment:
Yes.. we can try using the MARKER_INTERNAL for FWH. In fact, @sravya91 will likely give this a try asap. Everything looks good (thanks for the updates, Francisco). We'll merge this in asap so we can try out the surfaces.